### DISCUSSION TITLE
Support autodiscover filtering on BitBucket Cloud projects

### DISCUSSION THREAD

**Discussion by FHannes** [renovatebot/renovate#27093] [Suggest an Idea]
### Tell us more.

BitBucket Cloud supports organizing repositories in projects, which is are kind of like groups with additional access control features. It would be useful if Renovate could include or exclude repositories based on what project they are part of.

This feature seems to already be available for GitLab using the `autodiscoverNamespaces` configuration option? (Though it is not entirely clear to me if this allows you to also exclude namespaces instead of include them.)

BitBucket Cloud lacks quite a bit of functionality that the self-hosted platform and other platforms do include. Some of these features include archiving repositories and tagging. To archive projects on BitBucket Cloud, we move them into a specific project. That indicates that they are EOL or no longer relevant. Ideally we would want to exclude these from Renovate based on the project name instead of having to rename them or add a giant regular expression as an autodiscover filter.

**Comment by rarkins** [comment]
Converted to issue: https://github.com/renovatebot/renovate/issues/27661
